{"name": "Login valid user", "status": "failed", "statusDetails": {"message": "AssertionError: \nExpected: one of ('E', 'r', 'r', 'o', 'r', ' ', 'P', 'a', 'g', 'e')\n     but: was 'Welcome Page'\n\n", "trace": "  File \"c:\\users\\johnson_zhong\\pycharmprojects\\behavedemo\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\johnson_zhong\\pycharmprojects\\behavedemo\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps\\login.py\", line 22, in step_impl\n    assert_that (\"Welcome Page\", is_in(context.browser.title) )\n  File \"c:\\users\\johnson_zhong\\pycharmprojects\\behavedemo\\venv\\lib\\site-packages\\hamcrest\\core\\assert_that.py\", line 43, in assert_that\n    _assert_match(actual=arg1, matcher=arg2, reason=arg3)\n  File \"c:\\users\\johnson_zhong\\pycharmprojects\\behavedemo\\venv\\lib\\site-packages\\hamcrest\\core\\assert_that.py\", line 57, in _assert_match\n    raise AssertionError(description)\n"}, "steps": [{"name": "Given I navigate to login page", "status": "passed", "start": 1594102786103, "stop": 1594102786707}, {"name": "Then I login with \"abc\" and \"abcabc\"", "status": "passed", "start": 1594102786708, "stop": 1594102787619}, {"name": "Then I login Successfully", "status": "failed", "statusDetails": {"message": "AssertionError: \nExpected: one of ('E', 'r', 'r', 'o', 'r', ' ', 'P', 'a', 'g', 'e')\n     but: was 'Welcome Page'\n\n", "trace": "  File \"c:\\users\\johnson_zhong\\pycharmprojects\\behavedemo\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\johnson_zhong\\pycharmprojects\\behavedemo\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps\\login.py\", line 22, in step_impl\n    assert_that (\"Welcome Page\", is_in(context.browser.title) )\n  File \"c:\\users\\johnson_zhong\\pycharmprojects\\behavedemo\\venv\\lib\\site-packages\\hamcrest\\core\\assert_that.py\", line 43, in assert_that\n    _assert_match(actual=arg1, matcher=arg2, reason=arg3)\n  File \"c:\\users\\johnson_zhong\\pycharmprojects\\behavedemo\\venv\\lib\\site-packages\\hamcrest\\core\\assert_that.py\", line 57, in _assert_match\n    raise AssertionError(description)\n"}, "attachments": [{"name": "screenshot", "source": "1534be7d-e6ba-4d24-8ec7-ed6bb0ba003c-attachment.png", "type": "image/png"}], "start": 1594102787620, "stop": 1594102788026}], "start": 1594102782028, "stop": 1594102791910, "uuid": "ca26057e-41b4-4db4-8563-4a955f2f65e0", "historyId": "828847bd755f080eb1fdaf1ea16e8c0d", "labels": [{"name": "tag", "value": "login_fail"}, {"name": "tag", "value": "login"}, {"name": "severity", "value": "normal"}, {"name": "feature", "value": "Login & Logout"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}